### ISSUE TITLE
[Accessibility] Focus indicator missing on product thumbnail [Product page]

### ISSUE THREAD

**Issue by Manussakis** [woothemes/woocommerce#53015]
**Prerequisites (mark completed items with an [x]):** 
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:
The thumbnail image on the Product page doesn't have a visible focus indicator.
![Image](https://github.com/user-attachments/assets/59f68759-d2bc-4eab-a8b6-3e38642f1908)

### Component or Page:
Page: Product Page on Classic and Block-based themes

### Name affected element:
Image link.

### Relevant code snippets:
- https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/wc-template-functions.php#L1705

### Recommended fix:
Add `outline-offset: -1px;` to the link when it's focused.
